### PR TITLE
update for rock960c and android gpt images

### PR DESCRIPTION
**Pull request by vamrs**

### CODE REVIEW

**Comment by sdrobertw** [comment]
@ric96 can you please review this one?

File: consumer/rock/installation/linux-mac-rkdeveloptool.md [hunk 1]
```diff
@@ -71,20 +71,40 @@ Extract the image tar ball, for SD card/eMMC AIO image, the file list as below:
...
+- release maskrom key(important!)
 
-On the host PC, `lsusb` should show the following VID/PID if the board is in maskrom mode: `Bus 003 Device 061: ID 2207:0011`
+##### For ROCK960C
+You need USB3.0 or USB2.0 type A to type A male cable:
 
-**Note**: if no right USB device found, try:
+- plug the eMMC module on rock960c and power on
+- plug the rock960c to Linux desktop with USB type A to type A male cable from the **USB 3.0 OTG** port, as below:
+<img src="../rock960c/additional-docs/images/images-install/rock960c_maskrom.jpeg" data-canonical-src="../rock960c/additional-docs/images/images-install/rock960c_maskrom.jpeg" width="250" height="160" />
```
Comment (Mani-Sadhasivam) [inline consumer/rock/installation/linux-mac-rkdeveloptool.md]: @vamrs I think you haven't added this image to the PR!

Comment (Mani-Sadhasivam) [reply consumer/rock/installation/linux-mac-rkdeveloptool.md]: Rest looks good though

**Comment by vamrs** [comment]
The missing image is added now.

**Pull request merged**